src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -132,27 +141,36 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 					this.logService.error('Unexpected query event type "' + event.type + '" sent');
 					break;
 			}
-			self._cd.detectChanges();
+			//self._cd.detectChanges();
 		});
 
-		this.dataService.onAngularLoaded();
+		this.dataService.onComponentLoaded();
+	}
+
+
+	public render(container: HTMLElement): void {
+		this.nativeElement.style.width = '100%';
```

Comment:
any reason the styles are in CSS?

Reply:
This was in following with gridPanel.ts which used a similar feature for its render method

Reply:
it is apparently necessary to have the style and width be 100% else it won't show up

Reply:
Nevermind, apparently the size of the table is supposed to be handled by box-sizing which requires the div holding the slickgridcontainer to be renamed to match the css in editData.css (that was what the EDITDATA_SELECTOR was used for in the original Angular version)